pkg/sbom/io/encode.go

```diff
@@ -369,7 +377,7 @@ func (*Encoder) vulnerability(vuln types.DetectedVulnerability) core.Vulnerabili
 	return core.Vulnerability{
 		Vulnerability:    vuln.Vulnerability,
 		ID:               vuln.VulnerabilityID,
-		PkgID:            lo.Ternary(vuln.PkgID == "", fmt.Sprintf("%s@%s", vuln.PkgName, vuln.InstalledVersion), vuln.PkgID),
+		PkgID:            vuln.PkgID,
```

Comment:
We don't use `PkgID`. Perhaps we can remove this field.

Reply:
deleted https://github.com/aquasecurity/trivy/pull/7042/commits/06443ac2b794cc369be9543089b3ee834c2dfdb3